src/DynamoCoreWpf/UI/GuidedTour/GuideFlowEvents.cs

```diff
@@ -54,6 +57,17 @@ internal static void OnGuidedTourFinish(string name)
             }
         }
 
+        //Event that will be raised when the Guide is completely closed (this is specific for the Packages guide that allows the user to continue or exit the guide)
+        public static event GuidedTourClosedEventHandler GuidedTourClosed;
+        internal static void OnGuidedTourClosed(string name)
```

Comment:
Just to distinguish this event with the other one, we may want to name it `GuidedTourExited`, use Exited instead of Closed. I am still trying to confirm with UX why the exit experience would be different between tours at the same time

Reply:
@QilongTang renamed to GuidedTourExited in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13002/commits/b90cf210795f61b4e741bbefc029c13d1f4eda60